api/repositories/service_instance_repository.go

```diff
@@ -77,7 +77,27 @@ type PatchServiceInstanceMessage struct {
 	MetadataPatch
 }
 
-func (p PatchServiceInstanceMessage) Apply(cfServiceInstance *korifiv1alpha1.CFServiceInstance) {
+type PatchManagedSIMessage struct {
+	GUID        string
+	SpaceGUID   string
+	PlanGUID    string
```

Comment:
I wonder whether `PlanGUID` should not be a pointer? A patch may want to update the metadata, but on the plan